frontend/encore/simple-example.rst

```diff
@@ -139,7 +139,15 @@ may want also to :doc:`create a shared entry </frontend/encore/shared-entry>` fo
 Requiring CSS Files from JavaScript
 -----------------------------------
 
-You can also require CSS files from your JavaScript:
+Above, you created an entry called ``app`` that pointed to ``main.js``:
+
+.. code-block:: javascript
+
+    Encore
+        // ...
+        .addEntry('app', './assets/js/main.js')
```

Comment:
it's on the next line, and JavaScript will work just fine here :P 

**But don't leave a trailing comma for better diff readability!**, oh God no, we can't have that!! the Angels will the weep the blood of innocent, the sky will fall down like ashes, the seas will boil, the land will be engulfed with hordes of flies, chaos will rule the land for a 1000 years, nothing will ever grow anymore, all is dead. And.... ahhh, the compiler will fail to work 😑 JavaScript - forgives a missing semi-colon (when there is a space), but chokes on a trailing comma (because accepting that would be blasphemy).